tests/ttnn/python_api_testing/non_working_unit_tests/grayskull/test_argmax.py

```diff
@@ -49,15 +49,15 @@ def run_argmax_tests(
         raise e
 
     ttnn.close_device(device)
-
+    tt_result = tt_result.squeeze(2)
     assert len(tt_result.shape) == len(ref_value.shape)
     assert tt_result.shape == ref_value.shape
     assert_with_pcc(ref_value, tt_result, 0.99)
 
 
 test_sweep_args = [
     (
-        [(1, 3, 19, 98)],
+        [(1, 1, 19, 98)],
```

Comment:
Any reason we're getting rid of the old dims? Is the squeeze necessary/should it be two separate tests? IIRC argmax was brought up by Brian for Moreh and those dims may be important for them.

Reply:
this test wasn't working, hence under non_working_unit_tests